### DISCUSSION TITLE
Having issue with DS

### DISCUSSION THREAD

**Discussion by quayasf** [EmulatorJS/EmulatorJS#420] [Q&A]
Hi, I was using the codehelper to build a DS emulator that can run Pokemon Platinum but when I tried running it, I was told that the content was unable to load.  Is this a problem on my end, or is this how the emulator is supposed to work?  Thank you.

![image](https://user-images.githubusercontent.com/114786899/202607242-3efd8ce9-d229-46d5-8c72-80e64b1ffeb1.png)
![image](https://user-images.githubusercontent.com/114786899/202607293-9d16aedb-9b88-4f19-be4c-e5695fa568ff.png)

**Comment by ethanaobrien** [comment]
Could you please include a log of the console?

**Reply by quayasf** [reply]
Here is the console log
[consolelog.log](https://github.com/EmulatorJS/EmulatorJS/files/10037878/consolelog.log)

**Reply by ethanaobrien** [reply]
Try renaming your file and putting it in the code editor again

**Reply by quayasf** [reply]
Still having problems
![image](https://user-images.githubusercontent.com/114786899/202731958-4a2d4dd4-d97b-462d-9665-15b7688c7b26.png)
[consolelog2.log](https://github.com/EmulatorJS/EmulatorJS/files/10042229/consolelog2.log)

**Reply by ethanaobrien** [reply]
How come your file is 7ziped, yet the game url says the file extension is nds.

**Reply by quayasf** [reply]
idk, but i got the file in a compressed zip folder and extracted it using 7zip

**Reply by quayasf** [reply]
the file worked when i put it into the demo of emulatorjs

**Reply by ethanaobrien** [reply]
I don't know. This is probably a cache issue

**Reply by quayasf** [reply]
aight ty
